client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -92,6 +92,7 @@ export class DisplayTable extends React.Component {
           sum_func: (sum, value) => ... <- (function) Custom sum func. Default to sum + value
           sort_func: (a, b) => ... <- (function) Custom sort func. Default to _.sortBy
           sum_initial_value: 0 <- (number) Default to 0
+          color: "class name"
```

Comment:
This config can be a lot more flexible. With how it's written it is:
1. only limited to style color of td
2. only styles specific color
3. What if in the future, I want to use textGreen but it has nothing to do with `spending`?

You probably get my point. so this can be written with a lot more flexibility.

If you want to have additional config dedicated to `td` style, we can have, `td_class_name` (maybe think of a better name if that sounds bad). It would be a string, which is a css classname defined in where DisplayTable is called. That way, whoever is calling it will need to specify the color. Furthermore, it can be any css rule, not limited to only the color.

Reply:
BUT, I'd personally prefer to actually even not do that haha. For a reason where it's kind of a duplicate of `formatter` config, and I want to avoid cluttering up the configs and keep it cleaner as much as possible.

so with formatter, you can do something like:
`formatter: (value) => <Format style={{color: 'red'}} type={"dollar"} content={value} />`
okay but then `Format` probably won't take style attribute so maybe you can add that, or do some workaround!